From 895ce71e33e43f6006f5490b784214c96780bfda Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Mon, 13 Aug 2018 15:15:27 +0000
Subject: [PATCH] OS-7117 deliver uefi-rom.bin for bhyve Reviewed by: Jorge
 Schrauwen <jorge@blackdot.be> Reviewed by: Hans Rosenfeld
 <hans.rosenfeld@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 overlay/generic/manifest                      |   1 +
 .../generic/usr/share/bhyve/uefi-csm-rom.bin  | Bin 2097152 -> 1048576 bytes
 overlay/generic/usr/share/bhyve/uefi-rom.bin  | Bin 0 -> 1048576 bytes
 3 files changed, 1 insertion(+)
 mode change 100755 => 100644 overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
 create mode 100644 overlay/generic/usr/share/bhyve/uefi-rom.bin

diff --git a/overlay/generic/manifest b/overlay/generic/manifest
index d8b88125..ebc1a35c 100644
--- a/overlay/generic/manifest
+++ b/overlay/generic/manifest
@@ -177,6 +177,7 @@ f usr/sbin/zonememstat 0555 root bin
 f usr/sbin/zonemon 0555 root bin
 d usr/share/bhyve 0555 root bin
 f usr/share/bhyve/uefi-csm-rom.bin 0444 root bin
+f usr/share/bhyve/uefi-rom.bin 0444 root bin
 f usr/share/man/man1/json.1 0444 root bin
 f usr/share/man/man1m/fssstat.1m 0444 root bin
 d var/db 0755 root sys
diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
old mode 100755
new mode 100644
index 0cbe839c..0687c892
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
diff --git a/overlay/generic/usr/share/bhyve/uefi-rom.bin b/overlay/generic/usr/share/bhyve/uefi-rom.bin
new file mode 100644
index 00000000..cfefe0be
Binary files /dev/null and b/overlay/generic/usr/share/bhyve/uefi-rom.bin differ
-- 
2.21.0

